spec/visual/samples/livescript

```diff
@@ -0,0 +1,1441 @@
+/*
```

Comment:
Is it necessary to have such a long sample? It's great to give the lexer a workout but from a maintainability perspective, having a more focused sample tends to be more helpful. Could sections that are duplicative of syntax in other areas be deleted?

Reply:
Sure, I hope it's better now.